sensorprocess/sensorprocess.go

```diff
@@ -147,19 +179,37 @@ func tryAddLidarReadingUntilSuccess(ctx context.Context, reading []byte, reading
 
 // tryAddLidarReading adds a reading to the carto facade and does not retry (online).
 func tryAddLidarReading(ctx context.Context, reading []byte, readingTime time.Time, config Config) int {
-	startTime := time.Now()
+	startTime := time.Now().UTC()
```

Comment:
Does the replay sensor also provide UTC?

Reply:
just checked and saw that it did, was this causing issues?

Reply:
Not too much as this start time is just used to calculate time elapsed which is independent of timezone. Other instances of time not being cast to UTC did cause issues so I decided to use it everywhere. However, I am happy to revert if desired.